Clarify that BURI does not imply burial (#311)
FamilySearch/GEDCOM: specification/gedcom-03-datamodel.md

`BASM` | Bas Mitzvah<br/>`g7:BASM` | The ceremonial event held when a Jewish girl reaches age 13, also known as "Bat Mitzvah."
`BIRT` | birth<br/>`g7:BIRT` | Entering into life.
`BLES` | blessing<br/>`g7:BLES` | Bestowing divine care or intercession. Sometimes given in connection with a naming ceremony.

> `BURI` | disposal of remains<br/>`g7:BURI` | Disposing of the mortal remains of a deceased person.

Author: ```suggestion
`BURI` | depositing remains<br/>`g7:BURI` | Depositing the mortal remains of a deceased person.
```

`CENS` | census<br/>`g7:INDI-CENS` | Periodic count of the population for a designated locality, such as a national or state census.
`CHR` | christening<br/>`g7:CHR` | Baptism or naming events for a child.
`CHRA` | adult christening<br/>`g7:CHRA` | Baptism or naming events for an adult person.
`CONF` | confirmation<br/>`g7:CONF` | Conferring full church membership.

> `CREM` | cremation<br/>`g7:CREM` | Disposal of the remains of a person's body by fire; a specific type of `BURI`.

Author: ```suggestion
`CREM` | cremation<br/>`g7:CREM` | The act of reducing a dead body to ashes by fire; a specific type of `BURI`.
```

An [Individual Event](#individual-events).
See also `INDIVIDUAL_EVENT_STRUCTURE`.

> #### `BURI` (Disposal of remains) `g7:BURI`

Author: ```suggestion
#### `BURI` (Depositing remains) `g7:BURI`
```

An [Individual Event](#individual-events).
See also `INDIVIDUAL_EVENT_STRUCTURE`.

> Although defined as any disposal of remains since it was introduced in the first version of GEDCOM, this tag is a shortened form of the English word "burial" and has been interpreted to mean "disposal of remains by burial" by some applications and users.
> In the absence of a clarifying `TYPE` substructure it is likely, but not guaranteed, that a `BURI` structure refers to a burial rather than another form of disposal of remains.

Author: ```suggestion
Although defined as any depositing of remains since it was introduced in the first version of GEDCOM, this tag is a shortened form of the English word "burial" and has been interpreted to mean "depositing of remains by burial" by some applications and users.
In the absence of a clarifying `TYPE` substructure it is likely, but not guaranteed, that a `BURI` structure refers to a burial rather than another form of depositing remains.

```

An [Individual Event](#individual-events).
See also `INDIVIDUAL_EVENT_STRUCTURE`.

#### `BURI` (Disposal of remains) `g7:BURI`

An [Individual Event](#individual-events).
See also `INDIVIDUAL_EVENT_STRUCTURE`.

> Although defined as any disposal of remains since it was introduced in the first version of GEDCOM, this tag is a shortened form of the English word "burial" and has been interpreted to mean "disposal of remains by placing under the ground" by some applications and users.
> In the absence of a clarifying `TYPE` substructure it is likely, but not guaranteed, that a `BURI` structure refers to a burial rather than another form of disposal of remains.
> 
> See also `CREM`, a speicifc subtype of `BURI`.
> 

Author: ```suggestion
Although defined as any disposal of remains since it was introduced in the first version of GEDCOM, this tag is a shortened form of the English word "burial" and has been interpreted to mean "disposal of remains by burial" by some applications and users.
In the absence of a clarifying `TYPE` substructure it is likely, but not guaranteed, that a `BURI` structure refers to a burial rather than another form of disposal of remains.
```
